test/controllers/individual/deceased/CheckDetailsControllerSpec.scala

```diff
@@ -240,6 +357,31 @@ class CheckDetailsControllerSpec extends SpecBase with MockitoSugar with ScalaFu
       application.stop()
     }
 
+    "redirect to the 'add a settlor' page when submitted if there are no other settlors but user selected yes to Are there any additional settlors for the trust?" in {
```

Comment:
What happens when you go in, select yes for are there any settlors, click continue and go back to task list and then go back in